cob_bms_driver/src/power_state_aggregator.py

```diff
@@ -63,11 +71,26 @@ def calculate_power_consumption(self):
 
     def calculate_relative_remaining_capacity(self):
         if self.full_charge_capacity != None and self.remaining_capacity != None:
-            return round(100.0*(self.remaining_capacity/self.full_charge_capacity), 3)
```

Comment:
please keep this line as a comment because this is the right calculation as soon as the bms delivers correct data